slither.config.json

```diff
@@ -1,5 +1,6 @@
 {
     "solc_remaps": "@=node_modules/@",
     "filter_paths": "(mocks/|node_modules/)",
+    "detectors_to_exclude": "pragma,solc-version,assembly,too-many-digits",
```

Comment:
I think it's better to ignore them in the code when necessary to avoid side-effects.
Except maybe `pragma`

Reply:
Done